rest-api/src/main/java/gov/cms/qpp/conversion/api/controllers/v1/ExceptionHandlerControllerV1.java

```diff
@@ -74,6 +75,23 @@
 
 		return new ResponseEntity<>(exception.getMessage(), httpHeaders, HttpStatus.NOT_FOUND);
 	}
+
+	/**
+	 * "Catch" the {@link NoFileInDatabaseException}.
+	 * Return the {@link AllErrors} with an HTTP status 422.
+	 *
+	 * @param exception The NoFileInDatabaseException that was "caught".
+	 * @return The NoFileInDatabaseException message
```

Comment:
Many copy pasta errors in this JavaDoc.  There are some JavaDoc errors also on the `handleFileNotFoundException` method above.

Reply:
😞